testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -577,4 +580,42 @@ public void testRenameTableAcrossSchema()
         assertFalse(getQueryRunner().tableExists(getSession(), tableName));
         assertFalse(getQueryRunner().tableExists(getSession(), renamedTable));
     }
+
+    @Test(dataProvider = "objectNamesProvider")
+    public void testCreateSchema(String name, boolean delimited)
+    {
+        if (!hasBehavior(SUPPORTS_CASE_SENSITIVE_IDENTIFIERS)) {
+            throw new SkipException("Skipping since case-sensitive-identifiers is not supported at all");
+        }
+        String schemaName = name;
+        if (delimited) {
+            schemaName = "\"" + name + "\"";
+        }
+        String canonicalizedSchemaName = getCanonicalizedName(name, delimited);
+        assertQueryFails("DROP SCHEMA " + schemaName, format("line 1:1: Schema '\\w+\\.%s' does not exist", canonicalizedSchemaName));
+        assertUpdate("CREATE SCHEMA " + schemaName);
```

Comment:
you are missing things tests for `ALTER SCHEMA` and `GRANT/REVOKE ... TO SCHEMA` as well as creating and using tables in given schema.